css/debug-bar.dev.css

```diff
@@ -319,6 +319,21 @@ body.debug-bar-maximized.debug-bar-visible {
 	vertical-align: top;
 }
 
+#querylist select#wpd-queries-sort-by {
+	padding: 2px;
+	line-height: 28px;
+	height: 28px;
+	vertical-align: middle;
+	margin: 1px;
+	font-size: 14px;
+	border-radius: 0;
+    border: 1px solid #ddd;
```

Comment:
Please us a one tab indent for this line and the lines below it.

Reply:
oops.
since this is a spacing issue, i shouldn't need to recompile the minified version, correct?

Reply:
Correct.

Reply:
oops.
since this is just a spacing issue, i shouldn't need to recompile the minified version, correct?